python/pyhit/__init__.py

```diff
@@ -3,28 +3,37 @@
 import subprocess
 import mooseutils
 
-moose_dir = mooseutils.git_root_dir(os.path.dirname(__file__))
-status = mooseutils.git_submodule_status(moose_dir)
+try:
+    import hit
```

Comment:
This is the important change, just try "import hit" do the fancy stuff if that fails.